packages/components/scss/BaseStyleSheet.scss

```diff
@@ -596,26 +597,26 @@ input[type='number']::-webkit-inner-spin-button {
 ::selection {
   //blink/webkit
   color: $foreground;
-  background: rgba($text-select-color, 0.99);
+  background: var(--dh-color-text-highlight);
```

Comment:
This doesn't seem to get mapped correctly in the derived monaco theme, and is appearing red. The value derived is `"hsla(222deg, 82.7%, 61.6%, 83%, 62%, 0.3)"` instead of the #rgba hex value.

![image](https://github.com/deephaven/web-client-ui/assets/4505624/516cd647-bc86-477a-ad81-ebdac81a5ae9)

Reply:
I like how monaco yells at you if you set an invalid color by making it red. Real useful for theming actually.

Reply:
Looks like I duplicated the S/L components here. Fixed